tests/epyccel/test_base.py

```diff
@@ -144,34 +144,39 @@ def test_same_string(language):
     test = epyccel_test(base.isnot_same_string, lang=language)
     test.compare_epyccel()
 
+@pytest.mark.parametrize( 'language', [
+        pytest.param("c", marks = pytest.mark.c),
+        pytest.param("fortran", marks = pytest.mark.fortran)
+    ]
+)
 def test_same_complex(language):
```

Comment:
Change to be removed

Reply:
[9fd2088](https://github.com/pyccel/pyccel-cuda/pull/32/commits/9fd20881c4c0f9b7560a391c1c3272609fa87308)